### DISCUSSION TITLE
Is there a config way to disable the API

### DISCUSSION THREAD

**Discussion by Obyka** [ihsn/nada#99] [General]
Hello,
I wasn't able to find something about how to disable the API in the documentation.
My initial idea was to disable the routes but is there a more appropriate way to do so ?

As we will be using the web interface, we would be glad if we could reduce the application scope.
Thanks in advance for your help,
Have a nice day

**Comment by mah0001** [comment]
@Obyka - there is no setting to disable API. For a temporary fix, you can try this:

edit file: `/application/libraries/MY_REST_Controller.php` and add the code below:

```
function __construct()
{
        show_404();
}
```

**Reply by mah0001** [reply]
see for a config option - https://github.com/ihsn/nada/commit/65279551f8d8d1fb482b5900b7f4c532075a03f3

**Reply by Obyka** [reply]
Thanks for the answer and the feature, it will be of great help :)
